chart/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: kube-httpcache
 description: Varnish on Kubernetes Helm Chart
-version: 0.8.0
-appVersion: v0.8.0
+version: 0.10.0
+appVersion: v0.10.0
```

Comment:
Raising the version manually is not necessary; it will be bumped by a bot user when a tag is released.

Reply:
```suggestion
version: 0.8.0
appVersion: v0.8.0
```